### DISCUSSION TITLE
BFG Admin Panel

### DISCUSSION THREAD

**Discussion by Xsaven** [HassanZahirnia/laravel-package-ocean#197] [Package Suggestions]
Hi! Please review my admin panel package for Laravel:

Documentation:
https://bfg-s.github.io/admin-documentation

Repository:
https://github.com/bfg-s/admin

Packagist:
https://packagist.org/packages/bfg/admin

**Comment by Xsaven** [comment]
@HassanZahirnia Tell me when packages are added?

**Comment by Xsaven** [comment]
@HassanZahirnia Why is my package not being considered at the moment? How can you ask for sponsor on the site if you are cutting off extensions and developers who could help you, but since you ignored my package and decided not to add it, I will not sponsor you!
![image](https://github.com/user-attachments/assets/662e1b46-3ccc-46b9-a510-d798848b3620)

**Reply by HassanZahirnia** [reply]
Hey! If you check the label description, you can see what I stated: "Not finding the package appealing to be added at this time"

A package has to be proven useful to the Laravel community for me to add them. I also have a "[How do we choose packages?](https://github.com/HassanZahirnia/laravel-package-ocean?tab=readme-ov-file#how-do-we-choose-packages)" section in the README file of the project, which I recommend for you to read.

About the sponsor part, I think you misunderstood the assignment here. There is no relation between the sponsorship and having a package added to the database. It's not a "Sponsor me = have your project added" situation.
The sponsoring is to support the maintenance of this project.

I appreciate your interest in having your package added here, but please understand that this is a "package suggestion" area and the process for approving a package is pretty opinionated. So please have no hard feelings 🌸🥰

Have a great day!
